torch/nn/utils/clip_grad.py

```diff
@@ -18,6 +20,9 @@ def clip_grad_norm_(parameters: _tensor_or_tensors, max_norm: float, norm_type:
         max_norm (float or int): max norm of the gradients
         norm_type (float or int): type of the used p-norm. Can be ``'inf'`` for
             infinity norm.
+        error_if_nonfinite (bool): if True, an error is thrown if the norm of
```

Comment:
Perhaps there's a better name for this arg. Any other suggestions?

Reply:
https://github.com/pytorch/pytorch/issues/46849#issuecomment-797053011

Reply:
I think the current name is fine, but the description of the parameter should be changed slightly.

In particular, I think it's possible for the norm of all individual gradients to be finite while the total norm is infinite (see the original example given by the issue reporter: https://github.com/pytorch/pytorch/issues/46849#issue-729701939).

It'd be more accurate to indicate that an error is thrown when the *total* norm is non-finite.